edi_project_oca/tests/test_project_listener.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo.addons.component.core import Component
+
+from .common import TestEdiProjectOcaCommon
+
+_logger = logging.getLogger(__name__)
+
+
+class TestProjectListener(TestEdiProjectOcaCommon):
```

Comment:
you don't need all this testing... these are std events provided by component_event. You can simply drop them :wink: